src/Metadata/Property/PropertyMetadata.php

```diff
@@ -60,7 +60,7 @@ public function getType()
      *
      * @param Type $type
      *
-     * @return self
+     * @return PropertyMetadata
```

Comment:
For me in this case (return value of the phpdoc), `self` and `this` would mean the exact same thing